src/components/books/book/book.css

```diff
@@ -1,3 +1,281 @@
+
+.btn-cont {
+  display: flex;
+}
+
 .book {
+  width: 100%;
+  display: flex;
+  justify-content: space-around;
+  background-color: #fff;
+  padding: 2rem;
+}
+
+.edit-book {
+  border: none;
+  background-color: #fff;
+  color: #0290ff;
+  font-size: 0.75rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  cursor: pointer;
+}
+
+.btn-cont > *:not(:last-child) {
+  margin-right: 1rem;
+  padding-right: 1rem;
+  border-right: 1px solid #0290ff;
+}
+
+.info-book-cont {
+  width: 50%;
+}
+
+.info-book-cont > *:not(:last-child) {
+  text-transform: capitalize;
+  margin-bottom: 0.5rem;
+}
+
+.category-book {
+  color: #888;
+}
+
+.title-book {
+  width: 12.313rem;
+  height: 1.813rem;
+  font-size: 1.4rem;
+  font-weight: bold;
+  font-stretch: normal;
+  text-transform: uppercase !important;
```

Comment:
[OPTIONAL] I recommend avoiding using !important statements while working with CSS. The reason for this is that is difficult to know if a rule is being overridden by these statements when working on other areas of our code. Which generally means a headache while looking at why something is not working correctly. [You can read more about the Best Practices of CSS here.](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)